package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-webpack-plugin",
-  "version": "0.10.5",
+  "version": "0.10.6",
```

Comment:
please don't modify the package version. project maintainers - those with publish ability - will take care of that. pretty much the standard on webpack-contrib projects (and from what I've gathered, most projects)